.env.example

```
@@ -1,4 +1,5 @@
 ENCRYPTION_SALT_ROUNDS=10
 JWT_SECRET=REPLACE_WITH_RANDOM_256_BIT_SECRET
 JWT_EXPIRES_IN=86400
-MONGODB_URI="DBLINK_HERE"
\ No newline at end of file
+MONGODB_URI="DBLINK_HERE"
+RESEND_API_KEY=REPLACE_WITH_YOUR_RESEND_API_KEY
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)